confluent_kafka/error.py

```diff
@@ -37,23 +33,49 @@ class ConsumeError(KafkaException):
 
     """
     def __init__(self, error_code, exception=None, message=None):
+        kafka_error = self.constant_to_kafka_error(error_code, exception)
```

Comment:
Why not call it error_code_to_kafka_error() since that is how it is used, and constant is sort of ambigious?